src/style.css

```
@@ -1 +1,93 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin-top: 0;
+  font-weight: 300;
+ }
+
+ .search-form--big, .search-form--small {
+  position: relative;
+ }
+
+ .search-form--big::before {
+  top: 45px;
+  left: 26px;
+  height: 19px;
+  width: 19px;
+}
+
+.search-form--small::before {
+  top: 15px ;
+  left: 13px;
+  height: 11px;
+  width: 11px;
+}
+
+ .search-form--big::before,
+ .search-form--small::before {
+   position: absolute;
+   background-image: url("./images/Search.svg");
+   background-size: contain;
+   background-repeat: no-repeat;
+   content: "";
+}
+
+.search-form__input {
+  width: 100%;
+  font-family: Avenir-Heavy, sans-serif;
```

```suggestion
  font-family: inherit;
```
Apply font-family for `html` tag selector